Dangerfile

```diff
@@ -32,7 +32,7 @@ end
 
 # Lint with Rubocop and report violations inline in GitHub
 github.dismiss_out_of_range_messages # This way, fixed violations should go
-renaming_map = (git.renamed_files || []).map { |e| [e[:before], e[:after]] }.to_h # when files are renamed, git.modified_files contains old name, not new one, so we need to do the convertion
+renaming_map = (git.renamed_files || []).to_h { |e| [e[:before], e[:after]] } # when files are renamed, git.modified_files contains old name, not new one, so we need to do the convertion
```

Comment:
The Rubocop docs say this about this change: 

> This cop is unsafe, as it can produce false positives if the receiver is not an Enumerable.

I believe the usage here is an enumerable, but please let me know if I'm wrong so that this isn't changed incorrectly.

Reply:
TIL that `Enumerable#to_h` could take a block, and thus be used directly instead of `Enumerable#map` followed by `#to_h`. Nice.

And yes, here `git.renamed_files || [])` is an `Enumerable` so we should be good.

Reply:
Thank you for double-checking my assumption 👍